utils/spdx/src/test/kotlin/model/SpdxDocumentTest.kt

```diff
@@ -72,13 +73,17 @@ class SpdxDocumentTest : WordSpec({
 
     "The official JSON example without ranges from the SPDX specification version 2.2" should {
         "have idempotent (de-)serialization" {
-            val json = spdxExamplesDir.resolve("SPDXJSONExample-v2.2-no-ranges.spdx.json")
-            val document = SpdxModelMapper.read<SpdxDocument>(json)
-
-            val serializedJson = SpdxModelMapper.toJson(document)
-            val deserializedJson = SpdxModelMapper.fromJson<SpdxDocument>(serializedJson)
+            val json = spdxExamplesDir.resolve("SPDXJSONExample-v2.2-no-ranges.spdx.json").readText()
+
+            val reSerializedJson = SpdxModelMapper.fromJson<SpdxDocument>(json).let {
+                SpdxModelMapper.toJson(it)
+            }.let {
+                SpdxModelMapper.fromJson<SpdxDocument>(it)
+            }.let {
+                SpdxModelMapper.toJson(it)
+            }
```

Comment:
Isn't this reserializing once too much? Note that `shouldEqualJson` will internally also deserialize once in order to do the comparison on JSON nodes.